lara-typescript/src/interactive-api-client/api.ts

```diff
@@ -388,3 +391,67 @@ export const getInteractiveSnapshot = (options: IGetInteractiveSnapshotOptions)
 export const getLibraryInteractiveList = (options: IGetLibraryInteractiveListRequest) => {
   THROW_NOT_IMPLEMENTED_YET("getLibraryInteractiveList");
 };
+
+export const writeAttachment = (params: Omit<IWriteAttachmentRequest, "requestId" | "operation">) => {
+  return new Promise<void>((resolve, reject) => {
+    const client = getClient();
+    const { name, content, options } = params;
+    const request: IAttachmentUrlRequest = { name, operation: "write", requestId: client.getNextRequestId() };
+    client.addListener("attachmentUrl", async (response: IAttachmentUrlResponse) => {
+      if (response.url) {
+        try {
+          await fetch(response.url, { ...options, method: "PUT", body: content });
+          resolve();
+        }
+        catch (e) {
+          reject(e);
+        }
+      }
+      else {
+        reject(new Error(response.error || "error writing attachment"));
+      }
+    }, request.requestId);
+    client.post("getAttachmentUrl", request);
+  });
+};
+
+export const readAttachment = (name: string) => {
```

Comment:
It seems good to explicitly define the return type here, since if it is changes code using this client will break.